Automattic/woocommerce-payments
# Generate RTL CSS assets

Fixes #6943

#### Changes proposed in this Pull Request
- Add `assets:rtl` NPM script to generate RTL versions of `assets/css/*` files using the already available `rtlcss` package, with the same JS configuration (default one).
- Update `postbuild:client` to also run `assets:rtl`, ensuring that we always have it updated on every build.
- Update remaining `WC_Payments_Utils::enqueue_style` to set `$has_rtl` to true.

More details about the proposed solution on paJDYF-cEC-p2

#### Testing instructions
- Remove `assets/css/*.rtl.css` file and run `npm run build:client`, the file should be successfully generated.
- Go to **WP Admin -> Settings** and switch language to Arabic (العربية).
- Go to **WP Admin -> Payments** and confirm that there is a stylesheet with ID `wcpay-admin-css-rtl-css` loaded.

-------------------


**Post merge**
